capabilities/code-maintainability.md

```diff
@@ -5,28 +5,47 @@ In the lifespan of a typical line of code, it will likely be written once, edite
+
+Every decision made during development — and every assumption about the software — has the potential to impact the long-term maintenance of the system.
+Therefore, it's important to weigh cost-benefit tradeoffs when making decisions and assumptions about the design and functionality of the software.
 
 ## Assessment
 
-To assess how mature your team or organization is in this capability, answer the questions below.
+To assess how mature your team or organization is in this capability, score yourself on the descriptions below.
+
+Don't worry if the description doesn't exactly match your situation. These descriptions are meant to be examples of situations that would qualify for the associated score. 
 
-Don't worry if the description doesn't exactly match your situation. The descriptions are meant as examples of situations that would qualify for the associated score. Generally, score yourself a 1 if this capability is completely missing from your team, a 2 if there is a lot of room for improvement, a 3 if there is some room for improvement, and a 4 if your team is exemplary in this capability.
+Generally, score yourself a 1 if this capability is completely missing from your team or organization, a 2 if there is a lot of room for improvement, a 3 if there is some room for improvement, and a 4 if your team or organization is exemplary in this capability.
+
+<ed> There are a few points of confusion here. First, we're asking the reader to score themselves on each description below. But, in the line above, we then refer to the description as "this capability". I suggest we avoid using the word "capability" since the one key capability we're focused on here is code maintainability. Perhaps these can be referred to as areas. But this leads to a second point of confusion: If we refer to something like "growing tech debt" as an "area" and we ask the reader to score themselves on it, then a score of 1 for "growing tech debt" means this area is completely missing from their team or organization...which doesn't really match up. I think two things need to happen: 1) edit "capability" to something like "area" and 2) clarify what each score means in the context of "areas". For example, a score of 4 for "proactive maintenance" means that your team or organization is exemplary in this area. I think the score of 1 probably needs most clarification. </ed> 
```

Comment:
We can replace "this capability" with Code Maintainability. I was trying to keep it generic, but I can see how that would lead to confusion since we're only talking about the Code Maintainability capability in this file.

Reply:
Done.